zad1/main.rb

```diff
@@ -1,13 +1,25 @@
 def sort_letters(string)
-
+	string.chars.sort.join
 end
 
 def count_vowels(string)
-
+	result = 0
+	string.each_char do |x|
```

Comment:
Można by się pozbyć zmiennej result, przez wykorzystanie metody `chars` której użyłaś powyżej i na zwróconym arrayu użyć metody [count](https://ruby-doc.org/core-2.5.3/Enumerable.html#method-i-count). Można zrobić to oczywiście jeszcze prościej używając metody `count`, która występuje analogicznie w klasie [String](https://ruby-doc.org/core-2.2.0/String.html#method-i-count)